### DISCUSSION TITLE
Code of Conduct

### DISCUSSION THREAD

**Discussion by Ovid** [Perl-Oshun/oshun#28] [Announcements]
[There is now a Code of Conduct](https://github.com/Perl-Oshun/oshun/blob/main/CODE_OF_CONDUCT.md) for Oshun.

Unfortunately, the only way of reporting is to send email directly to me. This doesn't scale.

* If the complaint is against me, I shouldn't be the one reviewing it
* There's no way to anonymously lodge a complaint
* If I'm busy and don't notice the complaint, it might not get acted on.

Suggestions welcome. I'm not very familiar with Github Organizations or how this is traditionally handled here. However, such a code is important.

@druud @happy-barney @perigrin @rabbiveesh @rwp0 @SysPete @tobyink @demerphq

**Comment by rwp0** [comment]
@Ovid 

Great, thanks!

Just FYI for group mentions you can use the feature below:

- https://github.blog/2012-05-09-introducing-team-mentions/
- https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team

Unfortunately I don't have enough organization access to modify it and create the team myself (and enhance it in many other ways). Can you grant that please?

**Comment by Ovid** [comment]
@rwp0 I've now added you as an admin. Thanks for offering :)

**Comment by rwp0** [comment]
Now we can mention the above people as @Perl-Oshun/core-team (if they accept the invitation)
